README.md

```diff
@@ -32,20 +32,30 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. What is the DOM?
 
```

Comment:
Can you please remove changes to the README?
😇 thx

Reply:
Whoops! Yeah, no problem.